serve.go

```diff
@@ -26,19 +26,22 @@ const (
 )
 
 type config struct {
-	Port int    `env:"VHS_PORT" envDefault:"1976"`
-	Host string `env:"VHS_HOST" envDefault:""`
-	Key  string `env:"VHS_KEY" envDefault:""`
-	GID  int    `env:"VHS_GID" envDefault:"0"`
-	UID  int    `env:"VHS_UID" envDefault:"0"`
+	Port               int    `env:"PORT" envDefault:"1976"`
+	Host               string `env:"HOST" envDefault:""`
```

Comment:
fwiw this should probably default to localhost only... otherwise the server is open by default with no auth...